tests/BasicPackets_Test.py

```diff
@@ -28,21 +23,19 @@ class two_int_field_packet(models.Packet):
         self.assertEqual(p.int_field, 12)
         self.assertEqual(p.int_field_signed, -12)
 
-
     def test_pkt_create_packet_with_default_field_value(self):
         class two_int_field_packet(models.Packet):
             int_field = models.IntField(default_val=12)
             int_field_signed = models.IntField(signed=True, default_val=-12)
 
-
         p = two_int_field_packet()
         self.assertEqual(p.int_field, 12)
         self.assertEqual(p.int_field_signed, -12)
 
     def test_pkt_create_class_from_bytes_string(self):
         """
-        This test verifies that a class can be created from a byte string and that the values 
-        are properly parsed.  
+        This test verifies that a class can be created from a byte string and
+        hat the values are properly parsed.
```

Comment:
hat --> that

Reply:
Also I personally set the char limit to 120 however the typical 100 pep8 is fine.